source/direct_debit/index.html.md.erb

```diff
@@ -5,248 +5,126 @@ weight: 80
 
 # Direct Debit
 
-Before reading this section, you should be familiar with [making
-payments](/making_payments). You should also
-have [integrated your service](/integrate_with_govuk_pay/#how-to-integrate-with-the-gov-uk-pay-api) with the GOV.UK Pay API.
+You can use Direct Debit if you want to take recurring payments. You can take the same amount or a different amount each time.
 
-## Synchronous and asynchronous payment methods
+You cannot automatically schedule payments. If you need to take regular payments, you could create your own scheduler to send API calls on a set schedule.
```

Comment:
Could this call out something more positive about this - e.g. You have flexibility about when you can take payments. You cannot automatically...

That frequency flexibility is a selling point!

Reply:
although i can see that further down we say 'you can collect at any interval'

Reply:
Thanks Katie - added "You can take payments whenever you need to" here to clarify.